Post/tests.py

```diff
@@ -1,3 +1,12 @@
-# from django.test import TestCase
+from Post.models import post
 
-# Create your tests here.
+
+class TestPost():
+
+    def test_post_creation(self):
+        post1 = post(nameOfPoster='Shoval',
```

Comment:
This time it seems not much needed since its a small amount of code, but later maybe divide the part of the setup (in this case it's the preparation of the strings for creating the Post object) to a separate function named something like "post_set_up()"  and only use it inside the actual test function.

Reply:
I do not really see any preparation of the setup here.
Do you mean to define the strings as variables/constants?

Reply:
I suggested in a more complicated tests to do so, maybe using also a teardown_function  and also setup_function- isn't it's recommended?

Reply:
Fist, with pytest you should use its [fixtures](https://docs.pytest.org/en/stable/fixture.html), not the [xunit style](https://docs.pytest.org/en/stable/xunit_setup.html).
A test format is usually something like: given setup, exercise action, assert result, cleanup setup.
Usually the "setup" is something more interesting than a string/name, so I would not consider constants as a setup. However, if the tests do have something interesting in common, constant may be defined at the module, class or any other level to show relationship or just express what this data represents.